stats.cpp

```diff
@@ -1,5 +1,21 @@
 #include "stats.h"
-
-Stats Statistics::ComputeStatistics(const std::vector<___>& ) {
-    //Implement statistics here
+#include <bits/stdc++.h>
+namespace Statistics {
+    
+    Stats ComputeStatistics(const std::vector<float>& v) {
+        //Implement statistics here
+        Stats obj;
+        auto n = v.size();
+        if(v.size() == 0)
+        {
+            obj.average = 0;
+            obj.max = 0;
+            obj.min = 0;
```

Comment:
0 is not the correct average, min or max of an empty array. The test for this code is also wrong. It is checking that the average of empty array is *not NaN.

Reply:
Updated the code. I am setting average, max and min to NAN if the vector is empty. Is this what you meant?